cmd/juju/interact/pollster.go

```diff
@@ -405,26 +405,44 @@ func (p *Pollster) queryAdditionalProps(vals map[string]interface{}, schema *jso
 		// (i.e. map key) is the "name" of the thing.
 		var name string
 		var err error
-		if schema.Default != nil {
+		// Note: here we check that schema.Default is empty as well.
+		if schema.Default == nil || schema.Default == "" {
```

Comment:
Still failing :-( with OS_REGION_NAME set.  perhaps check to see that no schema.EnvVars available too?